code/Test_definitions/device-reachability-status-subscriptions-retrieveDeviceReachabilityStatusSubscription.feature

```diff
@@ -15,7 +15,7 @@ Feature: Device Reachability Status Subscriptions API, vwip - Operation retrieve
   # References to OAS spec schemas refer to schemas specifies in device-reachability-status-subscriptions.yaml
 
   Background: Common Device Reachability Status Subscriptions setup
-    Given the resource "{apiroot}/device-reachability-status-subscriptions/vwip" as base-url
+    Given the resource "{apiroot}/device-reachability-status-subscriptions/v0.8/subscriptions" as base-url
```

Comment:
```suggestion
    Given an environment at "apiRoot"
    And the resource "/device-reachability-status-subscriptions/v0.8/subscriptions"
```

Reply:
Please can you also add this as the first entry under '# Implementation indications:' 

`# * apiRoot: API root of the server URL`